app/src/main/res/resource/values-de/strings.xml

```diff
@@ -56,20 +56,22 @@
     <string name="unsplash">Unsplash.com</string>
     <string name="about_unsplash">Freie (mach, was immer du willst) hochauflösende Fotos.</string>
```

Comment:
Yes, that's why I am asking this. I don't know what was intended/meant when writing the slogan…